dockstring/__init__.py

```diff
@@ -1,4 +1,9 @@
-from importlib.metadata import PackageNotFoundError, version
+try:
+    # Python >=3.8
+    from importlib.metadata import PackageNotFoundError, version  # pyright: ignore[reportMissingImports]
+except ModuleNotFoundError:
+    # Python 3.7
+    from importlib_metadata import PackageNotFoundError, version  # type: ignore[no-redef]
```

Comment:
```python
import sys
if sys.version_info >= (3, 8):
    from importlib.metadata import PackageNotFoundError, version 
else:
    from importlib_metadata import PackageNotFoundError, version
```

This might avoid some linting errors. It is the recommended import structure from mypy ([link](https://mypy.readthedocs.io/en/stable/common_issues.html#python-version-and-system-platform-checks))